backend/FwLite/MiniLcm/IMiniLcmWriteApi.cs

```diff
@@ -57,7 +57,7 @@ Task<WritingSystem> UpdateWritingSystem(WritingSystemId id,
     #endregion
 
     #region ExampleSentence
-    Task<ExampleSentence> CreateExampleSentence(Guid entryId, Guid senseId, ExampleSentence exampleSentence);
+    Task<ExampleSentence> CreateExampleSentence(Guid entryId, Guid senseId, ExampleSentence exampleSentence, BetweenPosition? position = null);
```

Comment:
could you document this method and explain where the sentence will go when position is null.